src/platform/plugins/shared/unified_search/public/dataview_picker/change_dataview.tsx

```diff
@@ -99,16 +103,18 @@ export function ChangeDataView({
 
   useEffect(() => {
     const fetchDataViews = async () => {
-      const savedDataViewRefs = savedDataViews
+      const availableDataViewRefs = savedDataViews
         ? savedDataViews
         : (await data.dataViews.getIdsWithTitle()) ?? [];
```

Comment:
Not really on you here, but should this be wrapped in a try catch? fyi @davismcphee